index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Page</title>
+</head>
+<body>
+    <h1 id="title">Microverse Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action="#" id="survey-form">
+        <label for="name" id="name-label">Write your name</label><br>
+        <input type="text" id="name" placeholder="Enter your name"><br>
+        <label for="email" id="email-label">Write your email</label><br>
+        <input type="email" id="email" value="email" placeholder="Enter your Email"><br>
+        <label for="number" id="number-label">Write your number</label><br>
+        <input type="number" id="number" min="0" max="9" placeholder="Enter a number"><br>
+        <label for="dropdown">Vehicle</label><br>
+        <select name="dropdown" id="dropdown">
+            <option value="vehicle">volvo</option>
+            <option value="vehicle">Mercedes</option>
+            <option value="vehicle">Jetta</option>
+        </select><br>
+        <label for="accessories">mouse</label><br>
+        <input type="radio" id="accessories" name="accessories" value="accessories"><br>
+        <label for="accessories">keboard</label><br>
+        <input type="radio" name="accessories" value="accessories"><br>
+        <label for="checkbox">countries</label><br>
+        <input type="checkbox" id="checkbox" value="checkbox"><br>
+        <label for="checkbox">Brazil</label><br>
+        <!-- <input type="checkbox" id="checkbox" value="checkbox"><br> -->
```

Comment:
Please do not leave unused code as comment inside the code base. It will make your codes unclean.